### PR TITLE
Migrate PHPUnit configuration to version 9

### PR DESCRIPTION
**Pull request by Nyholm**
This will take care of the warning about migrating configuration. 

Screenshot 2020-08-20 at 14 02 29

### CODE REVIEW

**Review by jderusse — APPROVED** [review]
looks good to me

**Comment by Nyholm** [comment]
Hm.. On PHP 7.x we are running PHPUnit 8. That gives a warning about config that is not recognized. 

I should make sure we are running PHP 9 as often as we can

**Comment by jderusse** [comment]
> I should make sure we are running PHP 9 as often as we can

that's the job of phpunit-bridge.

a solution would be to run `--migtate-confoguration` in CI before running the test.

**Comment by Nyholm** [comment]
Im more happy to get a warning about invalid config on PHP 7.2 than a warning about deprecated config on PHP8.0. 

I specify what version I want to the bridge.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - master
 
+env:
+  SYMFONY_PHPUNIT_VERSION: '9.3'
```
Comment (jderusse) [inline .github/workflows/ci.yml:10]: the phpunit-bridge should take care of this. I'm not comfortable with maintaining a hardcoded version in this repository :-/

Comment (Nyholm) [reply .github/workflows/ci.yml:10]: Okey. That is fair. 

Im not happy with rewriting config in CI =)

So, the question is: What warning do we like?
Option one: Deprecation warning (shown in the description)
Option two: A unknown config warning: 

Screenshot 2020-08-20 at 14 50 03


Im fine with whatever. When I started the PR I did not know of the "unknown config warning".

Comment (jderusse) [reply .github/workflows/ci.yml:10]: hmm "unknown config" is just a warning or a behavior is not enabled?
I guess it's about coverage, and given we don't use it, it's not a real issue.

I'm fine with beeing compatible with the last version (fix deprecation). But the lower version should work as expected.

Comment (Nyholm) [reply .github/workflows/ci.yml]: I agree, it is no real issue. 

I've updated the PR

**Comment by Nyholm** [comment]
PR is updated, No CI changes. 

But I did update .editorconfig to use indentation_size=2 on phpunit.xml.dist. (It seams that 2 is preferred by PHPUnit)

File: .editorconfig [hunk 2]
```diff
@@ -14,5 +14,5 @@ trim_trailing_whitespace = true
 [Makefile]
 indent_style = tab
 
-[src/**/Makefile]
-indent_style = tab
+[phpunit.xml.dist]
+indent_size = 2
```
Comment (jderusse) [inline .editorconfig:18]: good catch

**Review by jderusse — APPROVED** [review]

File: .editorconfig (hunk 2 above)
Comment (Nyholm) [reply .editorconfig:18]: Thank you

**Pull request merged**